src/adapters/configuration/adapter_config.py

```diff
@@ -499,6 +499,7 @@ class LoRAConfig(AdapterConfig):
     composition_mode: str = "add"
     init_weights: str = "lora"
     use_gating: bool = False
+    dtype: Optional[str] = None
```

Comment:
one minor request: could you add the dtype attribute to the python doc of the changed classes? thanks!